### PR TITLE
hotfix for auto stuff. please don't kill me

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
## About The Pull Request
Yes, I've got heartaches by the number. (fixes a blueprint and what bluepritn someone gets)

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
fix: I've got heartaches by the number. (gives NCR outlaw the proper blueprints)
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/wasteland.dm [hunk 1]
```diff
@@ -268,7 +268,7 @@ Raider
 		/obj/item/ammo_box/magazine/m10mm/adv/simple=2,
 		/obj/item/melee/onehanded/knife/bayonet = 1,
 		/obj/item/storage/box/ration/ranger_breakfast = 1,
-		/obj/item/book/granter/crafting_recipe/blueprint/m1carbine = 1)
+		/obj/item/book/granter/crafting_recipe/blueprint/r82 = 1)
```
Comment (Superlagg) [inline code/modules/jobs/job_types/wasteland.dm:271]: Are they supposed to get an r82 blueprint?

Comment (bearrrrrrrr) [reply code/modules/jobs/job_types/wasteland.dm:271]: FUCK FUCK FUCK FUCK FUCK FUCK FUCK

**Review by Tk420634 — APPROVED** [review]

File: code/modules/jobs/job_types/wasteland.dm (hunk 1 above)
Comment (Tk420634) [reply code/modules/jobs/job_types/wasteland.dm:271]: BEAR FIX OR I MERGE WITH YOUR FACE

Comment (bearrrrrrrr) [reply code/modules/jobs/job_types/wasteland.dm]: fixed

**Review by Superlagg — APPROVED** [review]

**Pull request merged**